src/features/defi/providers/foxy/components/FoxyManager/Withdraw/FoxyWithdraw.tsx

```diff
@@ -120,14 +120,22 @@ export const FoxyWithdraw = ({ api }: FoxyWithdrawProps) => {
       const returVal = bnOrZero(bn(gasPrice).times(gasLimit)).toFixed(0)
       return returVal
     } catch (error) {
-      // TODO: handle client side errors maybe add a toast?
       console.error('FoxyWithdraw:getWithdrawGasEstimate error:', error)
-      toast({
-        position: 'top-right',
-        description: translate('common.somethingWentWrongBody'),
-        title: translate('common.somethingWentWrong'),
-        status: 'error',
-      })
+      if (error instanceof Error && error.message.includes('Not enough funds in reserve')) {
+        toast({
+          position: 'top-right',
+          description: translate('defi.notEnoughFundsInReserve'),
+          title: translate('common.somethingWentWrong'),
+          status: 'error',
+        })
+      } else {
+        toast({
+          position: 'top-right',
+          description: translate('common.somethingWentWrong'),
+          title: translate('common.somethingWentWrong'),
+          status: 'error',
+        })
+      }
```

Comment:
I feel like this looks a bit too much duplicated, what about using a function `isNoEnoughFundsError(error)` containing the typeguard and using a ternary inside the description attribute?

It will be clearer to see that only the description change, correct me if I'm wrong!

Reply:
yea true, I think a function might be too much, but I'll make the description a ternary